src/map/packets.h

```diff
@@ -2299,6 +2299,16 @@ packet(0x96e,-1,clif->ackmergeitems);
 	// New Packets End
 #endif
 
+#if PACKETVER >= 20130320
+// new packets
+// changed packet sizes
+	packet(0x09a7,10,clif->pBankDeposit,2,4,6); // CZ_REQ_BANKING_DEPOSIT
```

Comment:
packets.h mean only packet added/changed in packets len table in client.
But is client using packet at add date or bit after no one know.
This mean for any changes like this need manual testing with clients.
I will left this to some one else, because in wine very low amount of clients working. i cant test them all.